code/mathematical-algorithms/catalan_number/README.md

```diff
@@ -1,4 +1,9 @@
+# Catalan Number
+The Catalan numbers are a sequence of natural numbers that have applications in the area of combinatorial mathematics.
+
+The firsts Catalan numbers are: 1, 1, 2, 5, 14, 42, 132, 429, 1430...
+
 # cosmos
```

Comment:
Remove this

Reply:
Do you have a suggestion on how to document it? From my understanding the algorithms under catalan_number/ just generate the Catalan numbers; so I though explaining a little bit would be appropriate.

Reply:
No, I meant remove the cosmos part. For the catalan numbers, just post a link to their [Wikipedia](https://en.wikipedia.org/wiki/Catalan_number).

Reply:
Oh right, hadn't noticed that. Thanks.